src/lib.rs

```diff
@@ -258,16 +252,19 @@ pub trait Domain {
     fn parse_note_plaintext_without_memo_ovk(
         &self,
         pk_d: &Self::DiversifiedTransmissionKey,
-        plaintext: &NotePlaintextBytes,
+        plaintext: &Self::CompactNotePlaintextBytes,
     ) -> Option<(Self::Note, Self::Recipient)>;
 
-    /// Extracts the memo field from the given note plaintext.
+    /// Splits the memo field from the given note plaintext.
```

Comment:
Paired with the method rename:
```suggestion
    /// Splits the given note plaintext into the compact part (containing the note) and
    /// the memo field.
```

Reply:
Fixed.